src/sentry/api/endpoints/release_thresholds/release_threshold_index.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import Q
+from django.http import HttpResponse
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.models.organization import Organization
+from sentry.models.release_threshold.release_threshold import ReleaseThreshold
+
+
+class ReleaseThresholdIndexGETSerializer(serializers.Serializer):
```

Comment:
we tend to call input serializers `Validators`. know we could standardize this naming better

Reply:
cool - i can update the name